### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by redmarverick**
- React project created
- Added Linters
- Created Form
- Created BooksList
- Created Navigation
- Created Books

**COMMANDS:
npm run build
npm run dev**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    setBooks(books.filter((book) => book.id !== id));
+  };
+
+  return (
+    <Router>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<BookList books={books} onDelete={handleDeleteBook} />} />
+        <Route path="/categories" element={<BookList books={books} onDelete={handleDeleteBook} />} />
+        <Route path="/add-book" element={<BookForm onAddBook={handleAddBook} />} />
```
Comment (emmiiorji) [inline src/App.jsx]: From the project requirements, the form to add new book isn't a separate page. Kindly let the form be in the booklist component.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @redmarverick, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Project was initialized using [Create React App](https://create-react-app.dev/)
- [React Router](https://github.com/remix-run/react-router/tree/dev/examples/basic) has been correctly set up


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.jsx [hunk 2]
```diff
@@ -23,9 +22,8 @@ const App = () => {
     <Router>
       <Navigation />
       <Routes>
-        <Route path="/" element={<BookList books={books} onDelete={handleDeleteBook} />} />
-        <Route path="/categories" element={<BookList books={books} onDelete={handleDeleteBook} />} />
-        <Route path="/add-book" element={<BookForm onAddBook={handleAddBook} />} />
+        <Route path="/" />
+        <Route path="/categories" element={<BookList books={books} onDelete={handleDeleteBook} onAddBook={handleAddBook} />} />
```
Comment (AdnanOlarmmi) [inline src/App.jsx]: - [ ] Hi, nice one moving the Form component as suggested by the former reviewer, please the routes should just be only two, the home route which has the Form component and the categories route which just has a text, So the Booklist and Bookform components should be in the same route and the categories on another route. Please take a look at [this repo](https://github.com/AdnanOlarmmi/Bookstore) for a better idea. 

image

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @redmarverick, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice move ahead.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Kindly add the proper commands in the `package.json` file.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @redmarverick       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Good implementation of the required changes👍 
- Descriptive readme👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**